third_party/cpgfunctionEP-0.3.2/CONTRIBUTING.md

```diff
@@ -0,0 +1,125 @@
+# Contributing to cpgfunction
```

Comment:
This is a contributing file for cpgfunctionEP that are good for new contributors and helpful for myself to look back at. This doesn't need to be included for E+, will be removed on the next push.